Doc/library/functions.rst

```diff
@@ -1916,10 +1916,20 @@ are always available.  They are listed here in alphabetical order.
         >>> list(zip(('a', 'b', 'c'), (1, 2, 3), strict=True))
         [('a', 1), ('b', 2), ('c', 3)]
 
-     Unlike the default behavior, it checks that the lengths of iterables are
-     identical, raising a :exc:`ValueError` if they aren't:
-
-        >>> list(zip(range(3), ['fee', 'fi', 'fo', 'fum'], strict=True))
+     Unlike the default behavior, it raises a :exc:`ValueError` if one iterable
+     is exhausted before the others:
+
+        ..
+           This doctest is disabled because doctest does not support capturing
+           output and exceptions in the same code unit.
+           https://github.com/python/cpython/issues/65382
+
+        >>> for item in zip(range(3), ['fee', 'fi', 'fo', 'fum'], strict=True):  # doctest: +SKIP
```

Comment:
Due to the comment the code block gets indented ([see preview](https://deploy-preview-100103--python-cpython-preview.netlify.app/library/functions.html#zip)). Not too sure how to fix it while keeping the comment before the code block, but if it's moved elsewhere (say after) then it works.

Reply:
Oh, I guess I checked the preview too soon after my push then since it was looking fine at the time. Interesting.
Maybe dedenting the comment would work, but that seems awkward. I'll try moving it to below the code block.

Reply:
Negative, the code block still gets indented. :-/

Reply:
Only dedenting the comment means it considers the code block part of the comment. Moving the comment below *and* dedenting it seems to do the trick.